eq-author/cypress/integration/e2e/e2e_piping_spec.js

```diff
@@ -0,0 +1,69 @@
+import { question, questionnaire, section } from "../../builders";
+import { get } from "lodash";
+describe("End to end piping", () => {
```

Comment:
Could we add some piping to the existing e2e and update the snapshot?

Reply:
Also we have a piping_spec - could it go there?

Reply:
Not quite as simple as updating and comparing the snapshot since the test pipes into the title and since the answerId being piped will be different between test runs, the deep equals comparison will fail unless we also ignore titles (which defeats the object of the test). It could live in the existing piping spec I guess, the only reason I separated it out was to distinguish between tests that only hit the API and tests that assert against publisher output.

Reply:
Thats a shame. I guess we would need a custom matcher to ignore the id in the piping but care about the rest of it.

Ok, for the moment can we just move it to piping as I'm afraid it would get lost here. I don't think I'm too bothered about whether we should make a great distinction based on the assertion. 

I kinda viewed the e2e spec as one test that would grow to encompass a large amount of author features to build a complex survey. But that hasn't really happened.